### ISSUE TITLE
log_level=0 does not completely disable logging

### ISSUE THREAD

**Issue by jschueller** [modelon-community/PyFMI#96]
currently pyfmi defaults to opening log file in the current dir by default when log_file_name=""
now I want to disable logs because the current directory is not writeable so I set log_level=0
but pyfmi still tries to open a log file
is this expected, or do we need another log level ?

cc @modelonrobinandersson @agnesramle, since you're working on streams

**Comment by modelonrobinandersson** [comment]
> currently pyfmi defaults to opening log file in the current dir by default when log_file_name=""
> now I want to disable logs because the current directory is not writeable so I set log_level=0
> but pyfmi still tries to open a log file
> is this expected, or do we need another log level ?
> 
> cc @modelonrobinandersson @agnesramle, since you're working on streams

Hi!

I believe that this is a bug, it seems that we do not take the log level into consideration before we try to open a log file.

**Comment by chria** [comment]
@jschueller  This is now fixed in 2.8.8

**Comment by jschueller** [comment]
seems it was reverted in 2.8.10, should I reopen ?

**Comment by chria** [comment]
Correct, It was reverted due to concerns that it might be confusing and cause regressions for users. We'll look at again in the near future so see what we can do to remove some of the concerns that was raised.
